### PR TITLE
Migrate annotate_ssm_motif_context

### PR DESCRIPTION
**Pull request by vladimirsouza**

### CODE REVIEW

File: R/get_coding_ssm.R [hunk 1]
```diff
@@ -12,7 +12,7 @@
 #' @param limit_pathology Supply this to restrict mutations to one pathology.
 #' @param limit_samples This parameter is deprecated, use `these_sample_ids` instead.
 #' @param these_samples_metadata Supply a metadata table to auto-subset the data to samples in that table before returning.
-#' @param these_sample_id Optional, restrict the returned maf to a set of sample IDs (this parameter is replacing the `limit_samples` parameter).
+#' @param these_sample_ids Optional, restrict the returned maf to a set of sample IDs (this parameter is replacing the `limit_samples` parameter).
```
Comment (mattssca) [inline R/get_coding_ssm.R:15]: Nice catch!

File: R/annotate_ssm_motif_context.R [hunk 2]
```diff
@@ -0,0 +1,191 @@
+#' @title Annotate mutations target AID motif
```
Comment (mattssca) [inline R/annotate_ssm_motif_context.R:1]: Also, this is **all** migrated code (i.e. no new code added to this function, right?)

**Review by mattssca — COMMENTED** [review]
Just one question about the migrated code. Thanks!

File: R/annotate_ssm_motif_context.R (hunk 2 above)
Comment (vladimirsouza) [reply R/annotate_ssm_motif_context.R:1]: Usually, I have different commits to migrate a function to a package and to make changes to it. In this case, it seems that I included the function and changed its examples in [this](https://github.com/morinlab/GAMBLR.results/pull/10/commits/2a05de242d3aec163b53445e4a947ee282b45cff) same commit (sorry, I see this is not a good practice). After that, I changed IRanges and GenomicRanges imports separately in [this](https://github.com/morinlab/GAMBLR.results/pull/10/commits/5a23ebf618fc36e12debca02779640637bc689ea) commit. 

Is there a better way to migrate a function to a package while letting reviewers know if the code was changed or not?

Comment (mattssca) [reply R/annotate_ssm_motif_context.R:1]: Thanks for clarifying, see the conversation in Slack gamblr-help for more info on how to deal with migrating functions in the future :)

**Review by mattssca — APPROVED** [review]
Thanks!

**Pull request merged**